src/SoilModel/boundary_conditions.jl

```diff
@@ -420,3 +487,167 @@ function boundary_fluxes(
     fϑ_l = vertical_flux(bc.hydrology, hydrology, X_cf, model, dz, face)
     return (fρe_int = fρe_int, fϑ_l = fϑ_l)
 end
+
+
+"""
+    function boundary_fluxes(
+        X,
+        bc::PrescribedAtmosForcing,
+        face::Symbol,
+        model::SoilModel,
+        cs,
+        t,
+    )
+
+Returns the boundary flux at the boundary `face` for all
+components of the soil model in the case of PrescribedAtmosForcing.
+
+`X` is a FieldVector with components `ϑ_l`, `θ_i`, and `T`. 
+The `bc` field contains the boundary condition
+types (`Dirichlet`, `VerticalFlux`, etc) for both the energy and hydrology equations.
+In the case of explicitly time-varying boundary conditions, we also must supply the current time
+`t`.
+
+The remaining fields are required in the case of certain `bc` types, such as Dirichlet.
+In this case, we are turning a state boundary condition into a flux boundary condition, and this
+requires knowing the side of the domain we are on (encoded in `face`), the values of `X` just interior
+to this face, and the center space `cs` itself.
+"""
+function boundary_fluxes(
+    X,
+    bc::PrescribedAtmosForcing,
+    face::Symbol,
+    model::SoilModel,
+    cs,
+    t,
+)
+    if face != :top
+        error(
+            "Prescribed atmosphere-driven boundary conditions are only valid at the top of the soil column.",
+        )
+    end
```

Comment:
```suggestion
function boundary_fluxes(
    X,
    bc::PrescribedAtmosForcing,
    ::Val{:top},
    model::SoilModel,
    cs,
    t,
)
```
You'd also need to call `Val` on the face symbol at the call site

Reply:
Im going to wait on this. In some of my functions, `face` is passed in but the actual value doesnt matter (but I specify the type = `Symbol`). 
I'm not 100% sure how to do Val{Symbol}, and also wondering if this could end up generating worse performance as in the warning at the end of this section https://docs.julialang.org/en/v1/manual/performance-tips/#man-performance-value-type